cuweb/rds
# WSDEV 3174 - TopBar

## PR Checklist & Notes

### What type of PR is this? (check all applicable)


### Related Tickets & Documents

- Jira Ticket / Github Issue #

### Is this a breaking change?

If yes, please note which project will be impacted.


Events Calendar

### Have you made an entry in the Changelog?

If not, please make an entry or your PR will be declined. Add new entries under the "Unreleased" section of the changelog.


### Have you made changes to dependencies?

If you've made changes to dependencies please note them here, adding if they've been added, removed or updated, including packages version where applicable.


"react-intersection-observer"

### QA Instructions, Screenshots, Recordings

Please replace this line with instructions on how to test your changes, and if applicable add a note
about any UI changes or if a test should be done on a specific browser. Add any other detail relevant to this PR.

Check menu is working correct when screen is resized.
